packages/react/src/themes/teams/components/Toolbar/toolbarVariables.ts

```diff
@@ -86,7 +78,6 @@ export default (siteVars: any): ToolbarVariables => ({
         borderFocus: siteVars.colorScheme.brand.borderFocus1,
         foregroundHover: siteVars.colorScheme.brand.foregroundHover,
         backgroundHover: 'transparent',
-        foregroundFocus: siteVars.colorScheme.brand.foregroundFocus,
```

Comment:
Why are you removing only this override? Shouldn't the `backgroundFocus` be removed too?